package.json

```diff
@@ -65,12 +65,13 @@
     "typescript": "^4.7.4"
   },
   "peerDependencies": {
-    "apollo-server-errors": "^2.5.0",
+    "apollo-server-errors": "^3.3.1",
     "graphql": "^14.7.0 || ^15.8.0"
   },
   "dependencies": {
+    "@graphql-tools/schema": "^7.1.5",
+    "@graphql-tools/utils": "^7.1.5",
     "graphql-tag": "^2.12.6",
```

Comment:
nit: you removed graphql-tools from `package.json` 2 commits after removing all of its usages.